### ISSUE TITLE
The links in the documentation don't work - link navigation does not work

### ISSUE THREAD

**Issue by UnlimitedChild** [RickStrahl/MarkdownMonster#757]
Hello,

link navigation does not work in any case, examples of archives can be found here:
https://github.com/patriciogonzalezvivo/thebookofshaders
https://github.com/MicrosoftDocs/cpp-docs

 Just open any MD file and try to follow the link.

Thanks for a great project!
Regards!

**Comment by RickStrahl** [comment]
You're gonna have to be more specific. What doesn't work? Just took a look at a few of the pages and it looks to me that browser links to other MD docs works as well as `#hash` navigation in documents.

**Comment by UnlimitedChild** [comment]
> You're gonna have to be more specific. What doesn't work? Just took a look at a few of the pages and it looks to me that browser links to other MD docs works as well as `#hash` navigation in documents.

This functionality does not work anywhere in my case =(

![2021-03-06 12_25_27-](https://user-images.githubusercontent.com/16466330/110205252-4d6b0780-7e77-11eb-943f-dc467841fa50.png)

**Comment by UnlimitedChild** [comment]
 Unable to render Markdown Document

An error occurred trying to parse the Markdown document to HTML. 

Error
Markdown elements in the input are too deeply nested - depth limit exceeded. Input is most likely not sensible or is a very large table. 
more info... 


Error Detail
at Markdig.Helpers.ThrowHelper.<CheckDepthLimit>g__DepthLimitExceeded|16_0()
at Markdig.Parsers.MarkdownParser.ProcessInlines()
at Markdig.Parsers.MarkdownParser.Parse()
at Markdig.Parsers.MarkdownParser.Parse(String text, MarkdownPipeline pipeline, MarkdownParserContext context)
at Markdig.Markdown.Parse(String markdown, MarkdownPipeline pipeline, MarkdownParserContext context)
at Markdig.Markdown.Convert(String markdown, IMarkdownRenderer renderer, MarkdownPipeline pipeline, MarkdownParserContext context)
at MarkdownMonster.MarkdownParserMarkdig.Parse(String markdown) in C:\projects\MarkdownMonster\MarkdownMonster\_Classes\MarkdownParser\MarkdownParserMarkDig.cs:line 75
at MarkdownMonster.MarkdownDocument.RenderHtml(String markdown, Boolean usePragmaLines, Boolean noBanner) in C:\projects\MarkdownMonster\MarkdownMonster\_Classes\MarkdownDocument.cs:line 


U:\cpp-docs-master\docs\assembler\masm\alias-masm -> Directives Reference

**Comment by UnlimitedChild** [comment]
![2021-03-06 12_25_27-1](https://user-images.githubusercontent.com/16466330/110205829-91134080-7e7a-11eb-9d44-f53bf935c3e0.png)

**Comment by RickStrahl** [comment]
I'm not sure what that broken document is supposed to accomplish:

https://github.com/MicrosoftDocs/cpp-docs/blob/master/docs/assembler/masm/directives-reference.md

Sure looks like this is using unsupported Markdown functionality that doesn't render on GitHub either. I'll check on the error (which would be in the MarkDig renderer), but it looks to me this doc has invalid Markdown.

Still not sure what you're saying about navigation not working. I can navigate all these links no problem.

I think the issue may be that you're navigating links on a network drive and it's a permissions issue. Two things you can try:

* Make sure you're using the Chromium Rendering (Toolbar blue Chromium icon)
* Move files to a non-network drive.


Please... keep your bug reports to one topic - if there's something else (like the non-render topic) start a new issue. It's hard to follow multiple issues in one thread here.

**Comment by RickStrahl** [comment]
I have no idea what you posted here. Sorry but this makes no sense to me.
